### DISCUSSION TITLE
米家灯组延迟问题

### DISCUSSION THREAD

**Discussion by erchuyi** [XiaoMi/ha_xiaomi_home#113] [General]
灯组增加到 ha 后状态更新延迟大

我发现在米家内，灯组的状态显示也有很大的延迟。

**可能的问题原因：** 
灯组内4个灯，手动按灯具实体开关，灯具同步打开，但米家 app 内单灯是逐个延迟更新开启状态，等到四个灯具全部打开后灯组才会显示开灯状态。

设备：易来筒灯    连接版本为 mesh1.0

**Comment by HoTK1** [comment]
你重新建灯组试试，我之前更新中枢以前也卡，后面重建了以后就好了，不管是速度还是同步

**Reply by erchuyi** [reply]
谢谢哥，我重新创建了灯组，但是还是有延迟，米家内创建的灯组开关灯，在 ha 里是昨天不同步的，延迟非常大，我现在用的是单灯接入 ha，然后在 ha 里创建灯组，在桥接至 homekit 使用，但这样有个弊端就是 通过控制 ha 内创建的灯组开关灯，每个灯不是统一开启/关闭的，是挨个开启/关闭
